Pages/InfoPages/Quads 1-4/Quads.aml

```diff
@@ -0,0 +1,25 @@
+<aml>
+
+<head>
+<title>Academic Complex</title>
+</head>
+
+<body>
+
+<!-- The building code is CNTR -->
+
+<h1>Basic Building Information</h1>
+<p> Location: The heart of the Polytechnic campus & west of the Student Union.
+
+    Uses: The building houses the campus Library, Barrett Honors College, Computing Commons, Career Preparation Center,
+    Changemaker Central, Student Success Center, & Writing center, along with state-of-the-art moderated classrooms.
+    The University Technology Office & American Indian Student Support Services also occupy offices here.
+
+</p>
+
+<h1>Another heading!</h1>
```

Comment:
Noticed that many of these files have this, to be clear that was there for testing purposes on my end, and does not need to be there.